src/components/FlawAffects/FlawAffectsTableRow.vue

```diff
@@ -327,7 +327,6 @@ function affectednessChange(event: Event, affect: ZodAffectType) {
 
 tr {
   height: 39.2px;
-  position: relative;
```

Comment:
Seems to be a problem on how the browser handles transitions and opacity, I've fixed it by removing the filter transition and hiding the semi-transparent button instead of disabling it